src/pages/Login/login.jsx

```diff
@@ -1,12 +1,55 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
 import { Header } from "../../Components/header/header";
 import { Sidebar } from "../../Components/sidebar/sidebar";
-
+import { useAuth } from "../../context/Auth";
+import "./login.css";
 export const LogInPage = () => {
+  const navigate = useNavigate();
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const { LoginHandler } = useAuth();
+  const guestHandler = () => {
+    setEmail("architsingh@gmail.com");
+    setPassword("architSingh123");
+  };

```

Comment:
You can name this as setGuestCredentials or something similar. This way it's understandable that what he function does. This is a personal suggestion you can keep this as well no issues.